src/main/webapp/app/exercises/programming/manage/update/update-components/custom-build-plans/programming-exercise-build-configuration/programming-exercise-build-configuration.component.ts

```
@@ -2,20 +1,80 @@
     timeoutChange = output<number>();
 
+    envVars: string = '';
+    isNetworkDisabled = false;
+
     isAeolus = input.required<boolean>();
 
     dockerImageField = viewChild<NgModel>('dockerImageField');
-
     timeoutField = viewChild<NgModel>('timeoutField');
+
+    isLanguageSupported: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)